src/pages/SignUpCharity/SignUpCharity.tsx

```diff
@@ -22,6 +27,28 @@ const SignUpCharity: FC = () => {
     setFormData([...removeOldValue, { field, value, section, page }]);
   };
 
+  const { data, isLoading, error } = useQuery({
+    queryKey: ['la'],
+    queryFn: async () => {
+      const { data } = await client.graphql<
+        GraphQLQuery<GetLocalAuthoritiesQuery & GetJoinRequestsQuery>
+      >({
+        query: getLocalAuthorities,
+      });
+
+      return data;
+    },
+  });
+
+  if (error) {
+    throw new Error();
```

Comment:
Should put a message in here. Failed to fetch LocalAuthorities data .etc

Reply:
thanks, resolved